modules/codegen/src/smithy4s/codegen/Renderable.scala

```diff
@@ -33,13 +33,56 @@ object Renderable {
 
   implicit val stringRenderable: Renderable[String] = (s: String) =>
     RenderResult(Set.empty, List(s))
-  implicit def listRenderable[A](implicit
-      A: Renderable[A]
-  ): Renderable[List[A]] = { (l: List[A]) =>
+  def typeRenderable(ns: String): Renderable[Type] = new Renderable[Type] {
```

Comment:
I think we could make this non-parameterised, and filter out the imports that could be removed "at the end of the world"